### PR TITLE
Contact view #14

### PR DESCRIPTION
**Pull request by gosiamaks**
Created subpage with contact
connect to #14

### CODE REVIEW

**Comment by gosiamaks** [comment]
connect to #14

File: app/views/contact/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+
+<div  class="mdl-grid ingrediets-cover lunch-gap--none grid--center">
```
Comment (wozane) [inline app/views/contact/index.html.erb:1]: drop empty lines

File: spec/helpers/contact_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# describe ContactHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe ContactHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (wozane) [inline spec/helpers/contact_helper_spec.rb]: this file can be dropped

File: spec/views/contact/index.html.erb_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "contact/index.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (wozane) [inline spec/views/contact/index.html.erb_spec.rb]: this file can be dropped

File: app/assets/stylesheets/contact.scss [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (wozane) [inline app/assets/stylesheets/contact.scss]: Is there any need for this file? It looks like it is completely empty.

File: app/assets/stylesheets/layout.scss.erb [hunk 5]
```diff
@@ -1,13 +1,43 @@
...
+}
+
+/*-- link --:*/
+.link {
+  font-size: 2em;
+}
+
+/* unvisited link */
+.link:link {
+    color: #000;
```
Comment (gotar) [inline app/assets/stylesheets/layout.scss.erb]: wrong indentation

File: app/assets/stylesheets/layout.scss.erb [hunk 6]
```diff
@@ -1,13 +1,43 @@
...
+}
+
+/* visited link */
+.link:visited {
+    color: #000;
```
Comment (gotar) [inline app/assets/stylesheets/layout.scss.erb]: wrong indentation

File: app/assets/stylesheets/layout.scss.erb [hunk 7]
```diff
@@ -1,13 +1,43 @@
...
+}
+
+/* mouse over link */
+.link:hover {
+    color: #fff;
```
Comment (gotar) [inline app/assets/stylesheets/layout.scss.erb]: wrong indentation

File: app/assets/stylesheets/layout.scss.erb [hunk 8]
```diff
@@ -1,13 +1,43 @@
...
+}
+
+/* selected link */
+.link:active {
+    color: #000;
```
Comment (gotar) [inline app/assets/stylesheets/layout.scss.erb]: wrong indentation

File: config/routes.rb [hunk 9]
```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
 
+  get 'contact' => 'contact#index'
+
```
Comment (gotar) [inline config/routes.rb]: drop empty line

**Comment by gotar** [comment]
+1

**Pull request merged**